modules/hal_nordic/Kconfig

```diff
@@ -205,6 +205,16 @@ endif
 
 endmenu # NRF_802154_SER_HOST || NRF_802154_SER_RADIO
 
+if (NRF_802154_RADIO_DRIVER || NRF_802154_SERIALIZATION)
```

Comment:
nit: `depends on` could be cleaner than the `if` - `endif` block

Reply:
Ok, changed